docs/ref/request-response.txt

```diff
@@ -610,6 +610,14 @@ Finally, you can pass ``HttpResponse`` an iterator rather than strings.
 ``HttpResponse`` will consume the iterator immediately, store its content as a
 string, and discard it.
 
+Closeable objects passed to a ``HttpResponse``,
```

Comment:
Are there any practical consequences to this behavior that make it worth documenting? If so, would be worth mentioning them I think.